pandas/io/formats/format.py

```diff
@@ -2103,11 +2103,11 @@ def __call__(self, num: float) -> str:
 
         if self.use_eng_prefix:
             prefix = self.ENG_PREFIXES[int_pow10]
+
```

Comment:
this is really minor, but could you please undo these unrelated newline changes? there's a few of them across the files you've modified, please check through them